apps/login/package.json

```diff
@@ -3,9 +3,10 @@
     "version": "1.0.0",
     "private": true,
     "scripts": {
-        "start:docker": "docker-scripts start",
+        "deploy:cd": "docker-scripts deploy",
         "build": "tools/getBluedotKeycloakTheme.sh",
-        "deploy:cd": "docker-scripts deploy"
+        "start": "docker-scripts start",
+        "test": "docker-scripts test"
```

Comment:
This makes the CI actually try to build the image on PRs, giving us more confidence in CI checks.

Other script edits make it easier to do local development